bookwyrm/migrations/0046_reviewrating.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 3.0.7 on 2021-02-25 18:36
+
+from django.db import migrations, models
+from django.db import connection
+from django.db.models import Q
+import django.db.models.deletion
+
+def convert_review_rating(app_registry, schema_editor):
+    ''' take rating type Reviews and conver them to ReviewRatings '''
+    db_alias = schema_editor.connection.alias
+
+    reviews = app_registry.get_model(
+        'bookwyrm', 'Review'
+    ).objects.using(db_alias).filter(
+        Q(content__isnull=True) | Q(content='')
+    )
+
+    with connection.cursor() as cursor:
+        for review in reviews:
+            cursor.execute('''
+INSERT INTO bookwyrm_reviewrating(review_ptr_id)
+SELECT status_ptr_id FROM bookwyrm_review
+WHERE status_ptr_id={:d}'''.format(review.id))
+
+def unconvert_review_rating(app_registry, schema_editor):
+    ''' undo the conversion from ratings back to reviews'''
+    # TODO: this does not work
```

Comment:
@cincodenada any chance you know how to write the reverse migrations for this? I gave up for the day

Reply:
An excellent TODO 😄 I do have some ideas here that I have not actually tried, so I will give this a go hopefully tonight, if not tomorrow morning 👍

Reply:
Okay, so I just opened #652 which is a PR from my repo into this PR, and...it's the silliest thing, but literally all we need to do is just let Django drop the table, since that table is the only thing that makes those records ReviewRatings instead of Reviews :shrug:

There's also a couple other things in there that I described better in the description :)